scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -371,6 +375,8 @@ class Router(formatOps: FormatOps) {
               if style.spaces.afterTripleEquals &&
                 t.tokens.map(_.syntax) == Seq("===") =>
             Space
+          case name: Term.Name if isDef && isSymbolicName(name.value) =>
```

Comment:
```scala
 case name: Term.Name if isSymbolicName(name.value) && isDefDef(owners(name))
```

Reply:
We should guard this behavior behind a configuration setting `spaces.afterSymbolicDefs`, see "Spaces.scala" and add an explanation for the configuration option in the docstring.

Reply:
Thanks for the feedback! Just a small question about the requested changes: what would be the correct way to get from `name: Term.Name` to the `Token` to call the `owners` function, something like `isDefDef(owners(formatToken.right))` ?

Reply:
`name.parent.exists(isDefDef)`